### ISSUE TITLE
Meta Information table in CMS should be marked as presentation

### ISSUE THREAD

**Issue by laflannery** [department-of-veterans-affairs/va.gov-cms#10154]
## Description
On the View screens in the CMS, the meta information is displayed in a table. It seems that the intention of this is for layout purposes in which case the table should be marked with `role="presentation"` to indicate to screen readers that the semantic markup of the table should be ignored.

## Screenshot
![image](https://user-images.githubusercontent.com/106678594/184136888-82afdfa1-1035-4a97-8865-ab451e9f26c6.png)

## Accessibility Standard

WCAG version 2.0 A, [Criterion 1.3.1](https://www.w3.org/WAI/WCAG21/Understanding/info-and-relationships.html)

## Acceptance Criteria
- [ ] Confirm if an issue already exists/is being tracked for the contrib modal
- [ ] Implement patch if it exists
- [ ] If patch is not available, add `role="presentation"` to `<table>` element
- [ ] Accessibility lead review
